client/sharkey_client.go

```diff
@@ -190,7 +194,10 @@ func (c *context) makeKnownHosts() {
 		log.Println(err)
 		return
 	}
-	raw, err := exec.Command("/usr/bin/sudo", "/bin/mv", tmp.Name(), c.conf.KnownHosts).CombinedOutput()
+	raw, err := exec.Command("/bin/mv", tmp.Name(), c.conf.KnownHosts).CombinedOutput()
```

Comment:
this code duplication feels error-prone.  I don't super-like it.  Maybe we should set up a Cmd struct ourselves, and prefix the sudo in?  If you drop by my desk tomorrow, I can show you an example in another project.